cdap-examples/LogAnalysis/src/main/java/co/cask/cdap/examples/loganalysis/ApacheAccessLog.java

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
The reference book has license statement. https://www.gitbook.com/book/databricks/databricks-spark-reference-applications/details

See bottom of the page.